### ISSUE TITLE
Desktop Sizing not persisting through restart [Question]

### ISSUE THREAD

**Issue by stal2k** [matzman666/OpenVR-AdvancedSettings#81]
This may have to do with where SteamVR/OVR-AdvancedSettings is located, but I recently purchased a laptop that is VR capable. Much to my surprise, the desktop size adjustments persist through reboot/restarting SteamVR, but not on my Desktop. 

Is this a quick fix, or something obvious I'm missing? If not I'd be happy to provide any relevant detail that is requested. Apologies if this has been covered, I scanned through the existing issues and turned up nothing.

**Comment by matzman666** [comment]
> Much to my surprise, the desktop size adjustments persist through reboot/restarting SteamVR, but not on my Desktop.

Advanced Settings does not persistently save the desktop overlay size. This is done on purpose because what I am doing is an ugly hack and I don't want to cause permanent problems. So when the desktop overlay size persists, SteamVR must be saving the size somewhere. Do you have different SteamVR versions installed on your laptop and on your desktop?

**Comment by stal2k** [comment]
No, not to my knowledge. I can say this has been observed across a few
updates of SteamVR (before and after home). I just double checked and both
machines are on the beta branch. I'd be happy to help track it down through
event logging or just regular logs if you let me know what to pull. To be
honest, I really enjoy the persistence. While I might be the exception, not
the rule on how much that quality of life feature is appreciated - it is.
I'll confess I haven't really poured through the source to any degree, but
if you're maintaining your own disk based configuration, would storing the
size values there and applying them at each startup of your app sidestep
the permanent problems?

Depending on your vision for Advanced settings, that might be a
proprietary(ish) value add via a branded desktop viewer. It might be out of
scope, but in 'theme' with making the overlay better. Cleaning up the
annoyances of the stock viewer i.e. size/readability, inability to click
apps launched with admin like XSplit or task manager and difficulty in
double clicking (BigScreen nails this) would be huge.
